src/Raven.Studio/typescript/models/database/tasks/periodicBackup/s3Settings.ts

```diff
@@ -35,6 +37,14 @@ class s3Settings extends amazonSettings {
                 this.testConnectionResult(null);
             }
         });
+
+        this.accessKeyPropertyName = ko.pureComputed(() => this.isBackBlaze() ? "Application Key ID" : "Access key");
+        this.secretKeyPropertyName = ko.pureComputed(() => this.isBackBlaze() ? "Application Key" : "Secret key");
+    }
+
+    private isBackBlaze() {
```

Comment:
can we move this function to single location and make static? Right now we have this logic in 2 places. I'm affraid we forgot to change second place during any update.

Reply:
done